src/java/org/apache/cassandra/db/marshal/ReversedType.java

```diff
@@ -60,6 +61,16 @@ private ReversedType(AbstractType<T> baseType)
         this.baseType = baseType;
     }
 
+    @Override
+    public ReversedType<T> overrideKeyspace(Function<String, String> overrideKeyspace)
+    {
+        AbstractType<T> newBaseType = baseType.overrideKeyspace(overrideKeyspace);
+        if (newBaseType == baseType)
+            return this;
+
+        return new ReversedType<T>(newBaseType);
```

Comment:
see https://github.com/datastax/cassandra/pull/331/files#r801303769, we should use `getInstance` to reduce GC.